governance/tech-lead-proposal-process.md

```diff
@@ -0,0 +1,79 @@
+# TAG App Delivery Technical Lead Proposal Process
+
+App Delivery Technical Leads operate as a team alongside the Co-Chairs. The Co-Chair
+team seeks enhance the leadership capabilities of the existing co-chairs across
+the key qualities of industry experience, hands-on cloud-native and security
+experience, as well as administrative experience needed to assist in the
+execution of the various projects and efforts that make the TAG App Delivery so
+successful.
+
+## Caveats of Technical Leads
+
+* The position of a technical lead does not guarantee future co-chair nomination
+* The position of a technical lead is not a paid position (co-chairs are also
+  not paid positions)
+* Technical leads [have responsibilities](roles.md) to the App Delivery TAG, the
+  leadership team, and the TOC
+* Nominations still
+  [require TOC
+  approval](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections)
+
+## Process of nominations
+
+Depending on the needs of the community, as determined by the App Delivery TAG
+leadership team, technical lead nominations may come in the form of "Nominees by co-chairs" at the moment.
+
+The App Delivery TAG does not have a current limit on the amount of technical leads.
+
+Final technical lead nominations will adhere to the [CNCF's TOC described
+process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+
+### Nominees by co-chairs
+
+Nominees by co-chairs adhere to the technical lead requirements described below.
+Co-chairs are responsible for documenting statistic information of all nominee
+types for endorsement and presentation to the TOC.
+
+## Technical Lead nomination requirements
+
+In order to verify nominees by the community or by co-chairs have experience
+with the group and are actively contributing in a meaningful way, the following
+requirements must be met in order for an individual to be nominated.
+
+If a given nominee does not meet all the requirements, the co-chairs will still
+review the nomination and justification to determine if enough intent and
+commitment is present to move forward with an endorsement.
+
+Nominees must:
+
+* be the author of at least one pull request against the TAG App Delivery repository
+* be active in the community within the last 6 months prior to their nomination.
+* active - Attending at least 1 TAG App Delivery Regular Meeting a month
+* engaging in community chat via slack or lists
+* commenting on PRs and issues to drive suggestions to proposals, define scope,
+  resolve clarity issues, etc.
+* agree to the nomination
+
+Nominations must:
+
+* come from someone other than the nominee
```

Comment:
I'm not sure this should be required - folks should be able to nominate themselves too. But I do believe they should be seconded by an existing leader.